cegs_portal/search/templates/search/v1/dna_feature.html

```diff
@@ -506,25 +525,57 @@ <h1>
         {% endfor %}
     ];
 
-    g("feature-dist-submit").onclick = function() {
-        let dist = Number(g("feature-dist").value);
-        let fetches = assemblies.map(a => {
-            let search_start = Math.max(a.start - dist, 0);
-            let search_end = a.end + dist;
-            return fetch(`/search/featureloc/${a.chromo}/${search_start}/${search_end}?assembly=${a.assembly}&search_type=overlap&accept=application/json`)
-                .then(response => {
-                    return response.json()
-                });
-        })
-        Promise.all(fetches)
-            .then(data => {
-                let features = [];
-                for (const d of data) {
-                    features.push(...d);
-                }
-                g("dnafeature").replaceWith(featureTable(features, "No nearby DNA Features"))
+    function fetchDataAndRenderTable() {
+    let dist = Number(g("feature-dist").value);
+    let fetches = assemblies.map(a => {
+        let search_start = Math.max(a.start - dist, 0);
+        let search_end = a.end + dist;
+        let url = `/search/featureloc/${a.chromo}/${search_start}/${search_end}?assembly=${a.assembly}&search_type=overlap&accept=application/json`;
+
+        let checkboxes = document.querySelectorAll('input[name="feature_type"]:checked');
+        checkboxes.forEach(checkbox => {
+            url += `&feature_type=${checkbox.id}`;
+        });
+
+        return fetch(url)
+            .then(response => {
+                return response.json()
             });
+    })
+    Promise.all(fetches)
+        .then(data => {
+            let features = [];
+            for (const d of data) {
+                features.push(...d);
+            }
+            let table = g("dnafeature");
+            while (table.firstChild) {
+                table.removeChild(table.firstChild);
+            }
+
+    if (features.length === 0) {
+
+    let parent = document.getElementById('tabs-find-features');
+    parent.className = "grid justify-items-center";
```

Comment:
Can you add these values to directly to the #tabs-find-features element instead of using javascript?